app/controllers/schedule_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class ScheduleController < ApplicationController
+  def save_scheduled_publishing_datetime
+    Document.transaction do
+      document = Document.with_current_edition.lock.find_by_param(params[:id])
+      @edition = document.current_edition
+
+      checker = Requirements::ScheduledDatetimeChecker.new(permitted_params)
+      issues = checker.pre_submit_issues
+
+      if issues.any?
+        flash[:issues] = issues.items_for(:scheduled_datetime)
```

Comment:
maybe we want to name this flash more specific to the form?

Reply:
I think we also need to set a flash with data of the fields they filled in so it can populate the form.